---
section: 02. RESEARCH METHODOLOGY
title: "ChIP Seq vs. CUT&RUN"
edate: 2019-12-16
tags:
---

ChIP sequencing has been a central technique on genome wide analyses and its quality is mainly dictated by the quality of the antibodies used in the analysis. The antibodies we have for you Yap/Taz have been validated in our hands and by others and are considered of good quality. One major drawback of ChIP sequencing method is the number of cells required to achieve high quality reads. While the number of distal epithelial cells isolated from an individual mouse could be sufficient for a ChIP seq experiment, the number of proximal epithelial cells is highly limited and would not suffice for ChIP. To address this limitation, a new method has been developed, Cleavage under targets and release using nuclease (CUT&RUN), that manages to release the targeted sequences that are cleaved directly but the attached nuclease for paired end DNA sequencing. CUT&RUN requires a very small number of cells as low as 1000 cells. As Yap/Taz do not directly bind to DNA, the antibody bound nuclease could end up far from the DNA to cleave it.

This work is done at the lab of Dr. Claudio Cantu at Linköping University who is a co-supervisor of my PhD studies. Dr. Cantu is an expert in mass spec proteomics, ChIPseq and CUT&RUN and he has applied similar workflows previously on Wnt/β-catenin signalling.